src/content/1.1/code/ocaml/snippet03.ml

```diff
@@ -1,9 +1,3 @@
-module Compose_Example
-    (F : Polymorphic_Function_F)
-    (G : Polymorphic_Function_G with type b = F.b) =
-struct
-  (** OCaml doesn't have a compose operator. So, creating one. **)
-  let ( >> ) g f x = g (f x)
```

Comment:
this was confusing because earlier F#'s chevron operator was mentioned which is the same `>>` operator but encodes _forward_ composition. I have no idea why these examples were so excessively verbose! It's like they were actively trying to give a bad impression of OCaml

Reply:
ok so I've read earlier issues that explain why it was this verbose. interesting POVs.